ces_container/build.ps1

```diff
@@ -0,0 +1,27 @@
+param (
+    [string]$imageName = "executor_container"
+)
+
+if (-not $imageName) {
+    Write-Host "Please provide an image name as an argument."
+    exit 1
+}
+
+$taskweaverPath = "..\taskweaver"
+$requirementsPath = "..\requirements.txt"
+
+if (Test-Path $taskweaverPath) {
+    Write-Host "Using local files from $taskweaverPath"
+    Copy-Item -Path $taskweaverPath -Destination ".\taskweaver" -Recurse
+    Copy-Item -Path $requirementsPath -Destination ".\requirements.txt"
+} else {
+    Write-Host "Local files not found."
+    exit 1
+}
```

Comment:
maybe we do not need to copy all these folders but only the folder needed as in Dockerfile? (is it possible to have no content copying to avoid changing the filed modification time and causing container layer invalidation).

Besides, maybe we could have those logics in Python to avoid duplicate them for bash and PowerShell.

Reply:
The Dockerfile can only reach the files in the same dir or in subdir. That is why I did the copy. If we can put the Dockerfile into the root dir of the project, this will not be a problem.

Changing to python is not urgent now. We can leave it for future work.

Reply:
change context dir